### ISSUE TITLE
Log user deletions with good user info

### ISSUE THREAD

**Issue by rfay** [warmshowers/Warmshowers.org#544]
It would be nice if we had a good log of user deletions. We should be able to do this with rules module, add a node to a running list.

**Comment by jackturner7678** [comment]
Yes....  It would help us on the helpdesk.

**Comment by kalpaitch** [comment]
Maybe [entity_delete_log](https://www.drupal.org/project/entity_delete_log)?

**Comment by rfay** [comment]
Yes, that looks promising. Could be done with just rules too, but
entity_delete_log probably provides a good place to actually *put* the info.
